platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/WizAssetsPlugin.java

```diff
@@ -67,24 +72,41 @@
     private static final int FILE_CREATION_ERROR = 7;
     private static final int JSON_CREATION_ERROR = 8;
 
-    private String pathToStorage;
+    private String pathToDatabase;
+    private String pathToAssets;
     private int blockSize;
 
     @Override
     public void initialize(CordovaInterface cordova, CordovaWebView webView) {
```

Comment:
Why not creating a specific `initialize` method to actually initialize when the `initialize` method is called in JS land?